datascience/tables.py

```diff
@@ -2824,8 +2915,53 @@ def as_html(self, max_rows=0):
         return '\n'.join(4 * indent * ' ' + text for indent, text in lines)
 
     def index_by(self, column_or_label):
-        """Return a dict keyed by values in a column that contains lists of
+        r"""Return a dict keyed by values in a column that contains lists of
         rows corresponding to each value.
+	
+	Args:
+	    ``columns_or_labels``: Name or label of a column of the Table,
+	    values of which are keys in the returned dict.
+	
+	Returns:
+	    A dictionary with values from the column specified in the argument
+	    ``columns_or_labels`` as keys. The corresponding data is a list of
+	    Row of values from the rest of the columns of the Table.
+	    
+	Examples:
```

Comment:
On this example, could you please the change the table so that there is one repeated value in one of the columns that we give index by? That would make it much more clear to users that the rows would be aggregated under that repeated value